tests/components/prometheus/test_init.py

```diff
@@ -1382,6 +1407,46 @@ async def input_number_fixture(
     return data
 
 
+@pytest.fixture(name="number_entities")
+async def number_fixture(
```

Comment:
Besides adding a fixture, you also need to add an actual test that verifies metric values, similar to the one we have for `input_number`:
https://github.com/home-assistant/core/blob/268425b5e3db07525635b06e3b0c1e2863ae513f/tests/components/prometheus/test_init.py#L272

Reply:
I believe I already did that, but it was in a different commit:

https://github.com/home-assistant/core/blob/72ad1ec74f9286c92d2481c953a05ef49867b8e9/tests/components/prometheus/test_init.py#L296-L317